src/AutorestSwift/View Models/OperationViewModel.swift

```diff
@@ -95,12 +110,40 @@ struct OperationViewModel {
 
         self.method = requests.first?.method
         self.path = requests.first?.path
+
+        if let headerValue = requests.first?.knownMediaType {
+            headerInitParams
+                .append(KeyValueViewModel(key: ".contentType", value: headerValue))
+        }
+
+        if let headerValue = requests.first?.mediaTypes?.first {
+            headerInitParams
+                .append(KeyValueViewModel(key: ".accept", value: headerValue))
+        }
+
         self.returnType = ReturnTypeViewModel(from: responses.first?.objectType ?? "")
 
         self.params = items
+        self.queryParams = queryParams
+        self.headerParams = headerParams
+        self.queryInitParams = queryInitParams
+        self.headerInitParams = headerInitParams
+        self.uriParams = uriParams
     }
 }
 
+private func isStandardHttpHeader(with headerName: String) -> HTTPHeaders? {
+    for header in HTTPHeaders.allCases where header.rawValue == headerName {
+        return header
+    }
+    return nil
+}
+
+private func convertHttpHeaderKey(from header: String?) -> String? {
```

Comment:
I'm not clear on what the purpose of this is. Are you trying to map something like `x-ms-examples` to `examples`?  It would probably be better to use a string initializer on the enum that will spit out the actual enum value. This becomes brittle if we have to change the enum names.